### ISSUE TITLE
where is the `onRemove` function

### ISSUE THREAD

**Issue by geminiyellow** [gor181/react-notification-system-redux#10]
hi @gor181 , i found this 

https://github.com/gor181/react-notification-system-redux/blob/90d3271d47abb257361cb15ae2bf9615d3eb3f19/src/notifications.js#L29-L31

you override the notification 's `onRemove` function, can you tell me how to use the `onRemove` in the options ?

i think 

```
onRemove: () => {
    this.context.store.dispatch(actions.hide(notification.uid));
}
```

should be 

```
onRemove: () => {
    this.context.store.dispatch(actions.hide(notification.uid));
    if(notification.onRemove) notification.OnRemove(notification);
}
```

**Comment by gor181** [comment]
hey @geminiyellow ,

You have a good point.
Thanks for the code, exactly what's missing.

Will try to ship this by EOD.

**Comment by geminiyellow** [comment]
thank you @gor181 , i use it in my project , so,  if you can release it asap, thank you very much.

**Comment by gor181** [comment]
hey @geminiyellow ,

It's published, version is `1.0.8`.
Try to pull it down from npm again

thanks
